### ISSUE TITLE
update 1.20.1 optifabric please!!!!

### ISSUE THREAD

**Issue by daunidebil** [Chocohead/OptiFabric#1239]
it breaks mine minecraft, and i can't fix it myself

**Comment by Poopooracoocoo** [comment]
How does it break? Please explain.

If you've got [tofus](https://english.stackexchange.com/questions/296505/where-is-tofu-for-font-fallback-box-glyph-coming-from) on Minecraft 1.19.4 or Minecraft 1.20.1, you'll need to use Fabric API 0.85.0 for now. #1123 tracks the incompatibility of OptiFabric with newer versions of FAPI

**Comment by daunidebil** [comment]
no, it sometimes launches a freezed minecraft window or loads on mojang screen but then stops at around 80%

**Comment by AuroraDoesCode** [comment]
Optifine Reflector issue, can you try the Actions build for https://github.com/Chocohead/OptiFabric/actions/runs/6316388932 please? Its related to CIT I believe, and when optifine tries to load these and customcolors.properties is where the crash occurs. I havent seen it with that actions build. It may cause other mod issues right now however.

**Comment by daunidebil** [comment]
> Optifine Reflector issue, can you try the Actions build for https://github.com/Chocohead/OptiFabric/actions/runs/6316388932 please? Its related to CIT I believe, and when optifine tries to load these and customcolors.properties is where the crash occurs. I havent seen it with that actions build. It may cause other mod issues right now however.

tried it, for every version except normal 1.13.25 fabric asks for some mod called mm with version 2.0

**Comment by AuroraDoesCode** [comment]
> > Optifine Reflector issue, can you try the Actions build for https://github.com/Chocohead/OptiFabric/actions/runs/6316388932 please? Its related to CIT I believe, and when optifine tries to load these and customcolors.properties is where the crash occurs. I havent seen it with that actions build. It may cause other mod issues right now however.
> 
> tried it, for every version except normal 1.13.25 fabric asks for some mod called mm with version 2.0

can you post a modlist or a log? MM is included with Optifabric.
